pages/wiki/openembedded.md

```diff
@@ -3,10 +3,243 @@ title: OpenEmbedded
 layout: documentation
 ---
 
-<p><a href="http://www.openembedded.org/wiki/Main_Page">OpenEmbedded</a> is the build system used by AsteroidOS, it allows to easily maintain embedded linux distributions. OE is also used by project such as WebOS-Ports, SHR or Ångström. OpenEmbedded is supported by the <a href="https://www.yoctoproject.org/">Yocto project</a> from the Linux Foundation and provides a rock-solid basis for AsteroidOS.</p>
-<p>OpenEmbedded uses the bitbake tool which is developed in python to generate images (like asteroid-image) from packages (like asteroid-launcher-dev) built from recipes (ex: asteroid-launcher_git.bb) provided in layers (ex: meta-asteroid).</p>
-<p>Layers are git repositories containing a bunch of related recipes, example of layers include board support packages like <a href="https://github.com/AsteroidOS/meta-dory-hybris">meta-dory-hybris</a> or meta-rockchip which describe a way to support a new machine, but also UI layers like meta-xfce or meta-gnome which describe the building process of graphic components.</p>
-<p>A recipe describe how to fetch, patch, configure, compile, install, package (in .rpm, .deb or .ipk) and test a piece of software. Bitbake handles all those operations and dependencies between them and between recipes. In the end it can generate images or SDK for multiple targets.</p>
-<p>Contributing to the AsteroidOS’s OE architecture mostly consists in maintaining the <a href="https://github.com/AsteroidOS/meta-asteroid">meta-asteroid</a> and meta-*-hybris repositories which contain the recipes related to Asteroid and the watches BSPs.</p>
-<p>In depth info can be found in <a href="https://docs.yoctoproject.org/index.html">the Yocto Project's manual</a>.</p>
+# OpenEmbedded
+
+[OpenEmbedded](http://www.openembedded.org/wiki/Main_Page) (OE) is the build system used by AsteroidOS. It provides a way to easily maintain embedded Linux distributions. OE is also used by projects such as WebOS-Ports, SHR or Ångström. OpenEmbedded is supported by the [Yocto project](https://www.yoctoproject.org/) from the Linux Foundation and provides a rock-solid basis for AsteroidOS.
+
+OpenEmbedded uses the [`bitbake`](https://docs.yoctoproject.org/bitbake/) tool which is developed in python to generate images (like asteroid-image) from packages (like asteroid-launcher-dev) built from recipes (ex: asteroid-launcher_git.bb) provided in layers (ex: meta-asteroid).
```

Comment:
For extra readability, it would be good to place all these technical references, like image names, package names, recipe names and layer names in backticks.

Reply:
done